requirements.txt

```diff
@@ -34,39 +37,3 @@ virtualenv-clone==0.5.7
 virtualenvwrapper==4.8.4
 watchfiles==0.15.0
 websockets==10.3
-
-bs4~=0.0.1
```

Comment:
With all these deletes, I'm guessing that one or both of us accidentally pip freezed while virtual env not active, maybe.